src/pages/messages/components/TicketDateIntervalFilter.tsx

```diff
@@ -25,22 +25,27 @@ export const TicketDateIntervalFilter = () => {
 
   const onChangeFromDate = (newDate: Date | null) => {
     const newFromDate = newDate ? formatDateStringToISO(newDate) : '';
-    if (!newFromDate && !selectedToDate) {
-      searchParams.delete(TicketSearchParam.CreatedDate);
-    } else {
-      searchParams.set(TicketSearchParam.CreatedDate, `${newFromDate},${selectedToDate}`);
+    // check if year has 4 digits
+    if (newFromDate.match(/^\d{4}-/)) {
```

Comment:
Kunne lagt dette i en egen funksjon over komponenten, så vi slipper å duplisere regexen
```suggestion
    if (hasValidYear(newFormDate)) {
```